.goreleaser.yaml

```diff
@@ -1,12 +1,16 @@
+version: 2
+
+project_name: secret-init
+
 dist: build/dist
 
+before:
+  hooks:
+    - go mod tidy
 builds:
-  - main: .
-    env:
+  - env:
       - CGO_ENABLED=0
-    flags:
-      - -trimpath
-    ldflags: "-s -w -X main.version={{ .Version }}"
```

Comment:
we should include this, otherwise we wont have version details in the built release

Reply:
imo it is better if is explicitly set since we need to rely on custom configs by goreleaser. this way we wont have to worry if things change on their end

Reply:
Agreed, I'm updating Goreleaser in all BV projects, where it is used.